migrations/versions/20220111_174117_create_categories_to_items_table.py

```diff
@@ -1,35 +0,0 @@
-"""create categories to items table
-
-Revision ID: a0a65e21f96f
-Revises: 4e8be240b7b1
-Create Date: 2022-01-11 17:41:17.976220
-
-"""
-from alembic import op
-import sqlalchemy as sa
-
-
-# revision identifiers, used by Alembic.
-revision = 'a0a65e21f96f'
-down_revision = '4e8be240b7b1'
-branch_labels = None
-depends_on = None
-
-
-def upgrade():
-    # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table('categories_to_items',
-    sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('item_id', sa.Integer(), nullable=False),
-    sa.Column('category_id', sa.Integer(), nullable=False),
-    sa.ForeignKeyConstraint(['category_id'], ['categories.id'], ),
-    sa.ForeignKeyConstraint(['item_id'], ['items.id'], ),
-    sa.PrimaryKeyConstraint('id')
-    )
-    # ### end Alembic commands ###
-
-
-def downgrade():
-    # ### commands auto generated by Alembic - please adjust! ###
-    op.drop_table('categories_to_items')
-    # ### end Alembic commands ###
```

Comment:
A migration should not be deleted if it's been merged to main. Please revert this